js/blocks/ExtrasBlocks.js

```diff
@@ -916,21 +916,21 @@ function setupExtrasBlocks() {
             let act = logo.blocks.activity ;
             let getFunc = {
```

Comment:
do these need to match the translated strings above? _()???

Reply:
yes , they need to be translated as well ,
 I have changed this to a switch statement ,please take a look